server/pairing/payload_management.go

```diff
@@ -79,6 +82,16 @@ func (ppm *AccountPayloadMarshaller) UnmarshalProtobuf(data []byte) error {
 	if pb.Multiaccount != nil {
 		ppm.multiaccountFromProtobuf(pb.Multiaccount)
 	}
+
+	if ppm.multiaccount.KeycardPairing != "" {
+		if common.IsMobilePlatform() {
```

Comment:
Would be nice to add some explanation for history, that we kept it as is, to avoid existing databases migrations